### DISCUSSION TITLE
What is KPS

### DISCUSSION THREAD

**Discussion by raparri01** [supabase/supabase#8728] [Questions] [answered]
The about and readme refer to running the admin api server on KPS but I cannot find what KPS stands for.

**Comment by steve-chavez** [comment, marked as answer]
https://github.com/supabase/supabase-admin-api used to be a private/internal repo.

KPS is an acronym composed of Kong PostgREST Supabase(Realtime), which were the only services Supabase ran on its first versions.

That's no longer accurate(since there are more services now) so it should be corrected(PRs welcome).
